JS/Lesson14/sinan/script.js

```diff
@@ -4,90 +4,119 @@
 const stringExercise8 = 'Algorithms are a fundamental part of computer science.';
-console.log("Exercise8: ", stringExercise8.split(" "));
+console.log(stringExercise8.split(" "));
+
 console.log('------------------------');
 // 9. Use the `join` method to join an array of strings into a single string and print it to the console.
 const arrayExercise9 = ["An", "array",  "of", "strings"];
-console.log("Exercise9: ", arrayExercise9.join("-"));
+console.log(arrayExercise9.join("-"));
+
+
 console.log('------------------------');
 // 10. Use the `replace` method to replace "can help" with "help" in the string stringExercise10 and print the result to the console.
 const stringExercise10 = "Software design patterns can help simplify complex can help systems.";
-console.log("Exercise10: ", stringExercise10.replace("can help", "help"));
+console.log('Exercises 10:',stringExercise10.replace("can help","help"));
+console.log('Exercises 10.1:',stringExercise10.replace(/ /g,"-"));
+
 
 // What's the best approach to find and replace all occurences?
 
 console.log('------------------------');
 // 11. Use the `startsWith` method to check if strings below starts with a price, and print the result to the console.
 const string1Exercise11 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise11 = "Bundle deal: Save $50 when you buy two products together";
-console.log("Exercise11: ", string1Exercise11.startsWith("$"));
-console.log("Exercise11: ", string2Exercise11.startsWith("$"));
+console.log('Exercises 11.1: ', string1Exercise11.startsWith("$"));
+console.log('Exercises 11.2: ', string2Exercise11.startsWith("$"));
+
 
 console.log('------------------------');
 // 12. Use the `endsWith` method to check if a string ends with a "!" (exclamation mark), and print the result to the console.
 const string1Exercise12 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise12 = "Bundle deal: Save $50 when you buy two products together!";
-
+console.log('Exercises 12.1:', string1Exercise12.endsWith('!'));
+console.log('Exercises 12.2:', string2Exercise12.endsWith('!'));
 
 console.log('------------------------');
 // 13. Use the `includes` method to check if a string includes a "%" character and print the result to the console.
 const string1Exercise13 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise13 = "Bundle deal: Save $50 when you buy two products together!";
+console.log('Exercises 13.1:', string1Exercise13.includes("%"));
+console.log('Exercises 13.2:', string2Exercise13.includes("%"));
 
 
 console.log('------------------------');
 // 14. Use the `substring` method to extract the word "Databases" from string stringExercise14 based on two indices and print it to the console.
 // Question: What is the difference between `substring()` and `slice()`?
 const stringExercise14 = 'Databases are an essential component of many applications.';
+console.log('Exercises 14:', stringExercise14.substring(9));
```

Comment:
This does the opposite of what's asked in the requirements. You need to print the word 'Databases' instead.